datadog_checks_base/tests/test_log.py

```diff
@@ -1,11 +1,19 @@
+# -*- coding: utf-8 -*-
+
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+
 from datadog_checks import log
 
 
 def test_get_py_loglevel():
-    """
-    Ensure function _get_py_loglevel is exposed
-    """
-    assert getattr(log, "_get_py_loglevel", None) is not None
+    # default value for invalid input
+    assert log._get_py_loglevel(None) == logging.INFO
+    # default value for invalid unicode input
+    assert log._get_py_loglevel(u'dèbùg') == logging.INFO
```

Comment:
AFAIK this is valid utf-8 input, so it's encoded properly. I guess we need so weirder string to test it?

Reply:
the intention was to test a proper encoding but into a key that's invalid